src/LuaTransformer.ts

```diff
@@ -471,6 +473,39 @@ export class LuaTransformer {
         }
     }
 
+    public transformImportEqualsDeclaration(declaration: ts.ImportEqualsDeclaration): StatementVisitResult {
+        const name = this.transformIdentifier(declaration.name);
+        const expression = this.transformModuleReference(declaration.moduleReference);
+
+        return tstl.createVariableDeclarationStatement(name, expression, declaration);
+    }
+
+    public transformModuleReference(moduleReference: ts.ModuleReference): ExpressionVisitResult {
```

Comment:
I wonder does it really make sense to expose it as a public visitor?
Also, have you considered to use `transformExpression` and just add new nodes there?

Reply:
I think it does, since these are just AST nodes being visited, even if we only know of one place where they are right now.